example/endpoints/accept-raw.ts

```diff
@@ -5,9 +5,10 @@ import { taggedEndpointsFactory } from "../factories";
 export const rawAcceptingEndpoint = taggedEndpointsFactory.build({
   method: "post",
   tag: "files",
-  input: ez
-    .raw() // requires to enable rawParser option in server config
-    .extend({}), // additional inputs, route params for example, if needed
+  // requires to enable rawParser option in server config:
+  input: ez.raw({
+    /* additional inputs, route params for example, if needed */
+  }),
```

Comment:
⚠️ this is breaking

Reply:
and there is an issue with type inference of `raw`

Reply:
UPDATE:
.extend issue was fixed in 3.23.7
https://github.com/colinhacks/zod/issues/3437

Reply:
fixed the issue

Reply:
migration described in 47308116